# Lists all active connections managed by the modem.

Header header

uint16[] tcp    # A list of open TCP ports
uint16[] udp    # A list of open UDP ports
